cli/tests/integrations/helpers/common.py

```diff
@@ -36,7 +36,7 @@ def exec_command(cmd, env=None, stdin=None):
 
     # This is needed to get rid of '\r' from Windows's lines endings.
     stdout, stderr = [std_stream.replace(b'\r', b'')
-                      for std_stream in process.communicate()]
+                      for std_stream in process.communicate(timeout=timeout)]
```

Comment:
Ideally, this would be split out into it's own separate commit. With the test change in its own commit as well.